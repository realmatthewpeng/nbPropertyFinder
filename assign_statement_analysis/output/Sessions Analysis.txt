Total number of lines found: 0


Incorrect Lines:


Precision = 0
Recall = 0.0

sessions['date'] = pd.to_datetime(sessions.date)
sessions = pd.read_csv('sessions.csv')
sessions = sessions.fillna(0)
arc_scores = sessions.apply(calc_arc_scores, axis=1)
sessions['hang_scores'] = hangboard_scores
power_scores = sessions.apply(calc_power_scores, axis=1)
climbing_scores = sessions.apply(calc_climbing_scores, axis=1)
sessions['climbing_scores'] = climbing_scores
sessions['power_scores'] = power_scores
sessions['gym_scores'] = gym_scores
tech_scores = sessions.apply(calc_technique_scores, axis=1)
sessions['scores'] = scores
sessions['arc_scores'] = arc_scores
sessions['yoga_scores'] = yoga_scores
gym_scores = sessions.apply(calc_gym_scores, axis=1)
scores = sessions.apply(calc_score, axis=1)
yoga_scores = sessions.apply(calc_yoga_scores, axis=1)
hangboard_scores = sessions.apply(calc_hangboard_scores, axis=1)
sessions['tech_scores'] = tech_scores
doripa = sessions[sessions.monkey == 'doripa']
sunshine_sessions = sessions[sessions.monkey == 'sunshine']
athlete['additional_index'] = x
athelete_gym_scores = athlete[athlete.gym_scores > 0]
athlete_hang_scores = athlete[athlete.hang > 0]
athlete_tech_scores = athlete[athlete.tech_scores > 0]
athlete_climbing_scores = athlete[athlete.climbing_scores > 0]
