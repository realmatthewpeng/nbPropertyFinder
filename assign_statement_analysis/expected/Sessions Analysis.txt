sessions = sessions.fillna(0)
sessions['date'] = pd.to_datetime(sessions.date)
sessions['scores'] = scores
sessions['hang_scores'] = hangboard_scores
sessions['gym_scores'] = gym_scores
sessions['climbing_scores'] = climbing_scores
sessions['arc_scores'] = arc_scores
sessions['tech_scores'] = tech_scores
sessions['power_scores'] = power_scores
sessions['yoga_scores'] = yoga_scores
doripa = sessions[sessions.monkey == 'doripa']
sunshine_sessions = sessions[sessions.monkey == 'sunshine']
athlete_tech_scores = athlete[athlete.tech_scores > 0]
athlete_hang_scores = athlete[athlete.hang > 0]
athlete_climbing_scores = athlete[athlete.climbing_scores > 0]
athelete_gym_scores = athlete[athlete.gym_scores > 0]
athlete['additional_index'] = x